conans/test/command/info_test.py

```diff
@@ -197,6 +197,16 @@ def create_export(test_deps, name):
         self.assertIn("{ from: 0, to: 1 }", html)
         self.assertIn("id: 0, label: 'Hello0/0.1@PROJECT'", html)
 
+    def graph_html_embedded_visj_test(self):
+        client = TestClient()
+        save(os.path.join(client.client_cache.conan_folder, "vis.min.js"), "")
+        save(os.path.join(client.client_cache.conan_folder, "vis.min.css"), "")
+        client.save({"conanfile.txt": ""})
+        client.run("info . --graph=file.html")
+        html = load(os.path.join(client.current_folder, "file.html"))
+        self.assertIn("<body>", html)
```

Comment:
I'd rather `assertIn` here for the actual paths of both *vis.min.js* and *vis.min.css*

Reply:
Ok